containers/kubernetes/troubleshooting/configuring-sql-storage-backend-helm.mdx

```diff
@@ -0,0 +1,134 @@
+---
+meta:
+  title: Configuring a SQL storage backend in Helm on Scaleway Kubernetes Kapsule
+  description: Troubleshoot issues deploying large Helm releases on Scaleway Kubernetes Kapsule by configuring a SQL storage backend.
+content:
+  h1: Configuring a SQL storage backend in Helm on Scaleway Kubernetes Kapsule
+  paragraph: Troubleshoot issues deploying large Helm releases on Scaleway Kubernetes Kapsule by configuring a SQL storage backend.
+tags: kapsule helm sql backend
+dates:
+  validation: 2024-11-22
+  posted: 2024-11-22
+categories:
+  - kubernetes
+---
+
+When deploying large Helm releases on Scaleway Kapsule Kubernetes, you might encounter errors like the following:
+
+```bash
+rpc error: code = ResourceExhausted desc = etcd quota exceeded: size 55000000/55000000, burst 424242/100000000
+```
+
+This happens because Helm stores release information in Kubernetes **Secrets** or **ConfigMaps**, which are limited to 1MB due to **etcd** storage constraints.
```

Comment:
```suggestion
This happens because Helm stores release information in Kubernetes **Secrets** or **ConfigMaps**, which are limited to 1 MB due to **etcd** storage constraints.
```